src/CurveLPOracle.sol

```diff
@@ -154,7 +161,7 @@ contract CurveLPOracle {
         return block.timestamp >= zph;
     }
 
-    function poke() external {
+    function poke() external payable {
```

Comment:
Not sure I like this. Perhaps is significant for something that is called very often, but in general I'd rather not do it to avoid confusion.
This case might be one where is worth it but not sure. If you think it should remain, what I'd do is at least putting a comment that it is strictly to save gas but DO not send ETH to it as it will be lost.

Reply:
Personally I think I'm fine with it being payable and adding a comment as @gbalabasquer suggested.
For using this as a common practice to save gas outside of an oracle poke we probably need a broader discussion.

Reply:
I'll add a comment but I think it's absolutely worthwhile here to save gas (also unlikely that the oracle team accidentally sends ETH).